Tests/LibMarkdown/TestImageSizeExtension.cpp

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Since we already have in infrastructure in `TestCommonmark` we could just write our own json files and run our test runner on it. This is probably outside the scope of this PR though.

Reply:
I did consider that, but it seemed a little trickier to get setup (since it'd need to load the json from somewhere too). 
Also, it seems like `TestCommonmark` is currently disabled, and I'm not sure what the best way to have it just run these tests would be.

Reply:
(Also writing HTML in C++'s raw string literals is easier than in json :^))